config-for-docker-build.js

```diff
@@ -11,6 +16,11 @@ exports.options = {
   'auto update': true,
   'session': true,
   'session store': 'mongo',
+  'session store options':{
+    autoRemoveInterval: process.env.KEYSTONE_SESSION_AUTO_REMOVE_INTERVAL || 10,
+    touchAfter: process.env.KEYSTONE_SESSION_TOUCH_AFTER || 0,
```

Comment:
In the documentation, it states `touchAfter` in `Lazy session update` section and says:

> `touchAfter: 24 * 3600` you are saying to the session be updated only one time in a period of 24 hours, does not matter how many request's are made.

Q: If we state `touchAfter`, it means we don't want to resave all the session on database every single time that the user refresh the page, then, what does `touchAfter: 0` means ?

Or it just a default value which will always be overwritten by `process.env.KEYSTONE_SESSION_TOUCH_AFTER` ?

Reply:
It is just a default value and should be used in development. I prefer to refresh the session during the common development.